azext_edge/edge/params.py

```diff
@@ -130,11 +130,14 @@ def load_iotops_arguments(self, _):
 
     with self.argument_context("iot ops support") as context:
         context.argument(
-            "ops_service",
+            "ops_services",
+            nargs="+",
+            action="append",
```

Comment:
personally, I think action="extend" is easier to work with in this scenario. The variable mapped to the param becomes one list, then we apply set to de-dupe it.

Reply:
ok will take a look!

Reply:
updated